portfolio/src/main/webapp/Gallery.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      </p>Location: Isabela, Puerto Rico.</p>
```

Comment:
So you'll want to make sure each of your HTML tags starts with an open tag `<>` and ends with a closing tag, `</>`
```suggestion
      <p>Location: Isabela, Puerto Rico.</p>
```

There's a couple of places here and in the `index.html` with similar issues. Maybe try an HTML validator?https://validator.w3.org/#validate_by_input

Reply:
Thank you for pointing out the HTML corrections I needed to make. I used the HTML validator you provided (link).

Corrections have been pushed to the existing pull request.